fv3core/tests/savepoint/conftest.py

```diff
@@ -99,12 +100,24 @@ def read_serialized_data(serializer, savepoint, variable):
     return data
 
 
-def process_grid_savepoint(serializer, grid_savepoint, rank, *, backend: str):
-    grid = make_grid(grid_savepoint, serializer, rank, backend=backend)
-    fv3core._config.set_grid(grid)
+def upper_grid(serializer, grid_savepoint, rank, layout, *, backend: str):
```

Comment:
What is an "upper" grid? Is there a reason for this function when you can directly call `make_grid`?

Reply:
It was originally going to be a pytest fixture, but you're right we can just call make_grid directly. Changing this.